src/services/PaymentsService.ts

```diff
@@ -0,0 +1,53 @@
+/**
+ * PaymentsService
+ * Provide services related to Payments (Nodo) to communicate with PagoPa and Backend API
+ */
+
+import { Either, left, right } from "fp-ts/lib/Either";
+import { PagamentiTelematiciPspNodoAsyncClient } from "italia-pagopa-api/dist/lib/clients";
+import {
+  InodoAttivaRPTInput,
+  InodoAttivaRPTOutput,
+  InodoVerificaRPTInput,
+  InodoVerificaRPTOutput
+} from "italia-pagopa-api/dist/wsdl-lib/PagamentiTelematiciPspNodoservice/PPTPort";
+
+/**
+ * Send a request to PagoPa to retrieve payment info (VerificaRPT)
+ * @param {InodoVerificaRPTInput} iNodoVerificaRPTInput - The request to send to PagoPa
+ * @param {PagoPaConfig} pagoPaConfig - Configuration about PagoPa WS to contact
+ * @return {Promise<Either<Error, InodoVerificaRPTOutput>>} The response provided by PagoPa as response
+ */
+export async function sendPaymentCheckRequestToPagoPa(
+  iNodoVerificaRPTInput: InodoVerificaRPTInput,
+  pagamentiTelematiciPSPNodoClient: PagamentiTelematiciPspNodoAsyncClient
+): Promise<Either<Error, InodoVerificaRPTOutput>> {
+  try {
+    const nodoVerificaRPT = await pagamentiTelematiciPSPNodoClient.nodoVerificaRPT(
```

Comment:
the ideal here would be to have nodoVerificaRPT() to return a `Promise<Either<>>` and to catch all exceptions.
if this was the case we could get rid of this function entirely : )
(but that's a pagopa-api duty I guess)